_posts/2017-04-01-crystal-0.22.0-released.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Crystal 0.22.0 released!
+summary: Crystal has a new development release.
+thumbnail: +
+author: straight-shoota
+release_notes: true
+---
+
+[Crystal 0.22.0](https://github.com/crystal-lang/crystal/releases/tag/0.22.0) has been released!
+
+As any release it includes numerous bugfixes, cool features and performance improvements.
+
+# Exciting Changes
+
+* Backtrace function names are read from DWARF sections.
+* Sigfault and exceptions are printed to STDERR.
+* `PP` and `p` return the printed value.
+* `XML::Node` supports setting and deleting attributes.
+* The `skip` macro method allows to skip a file: `{{ "{%" }} skip unless flag(:unix) %}`. No need to wrap entire files in `{{ "{%" }} if %}...{{ "{%" }} else %}` anymore!
```

Comment:
What are the `{{` `}}` for? (Same below)

Reply:
`{% ... %}` would be interpreted as Jekyll/Liquid template so it's just escaping that.

Reply:
Oh ok